cibyl/sources/elasticsearch/api.py

```diff
@@ -38,29 +38,39 @@
 class ElasticSearch(ServerSource):
     """Elasticsearch Source"""
 
-    def __init__(self, driver: str = 'elasticsearch',
+    def __init__(self: object, driver: str = 'elasticsearch',
                  name: str = "elasticsearch", priority: int = 0,
                  elastic_client: object = None,
                  enabled: bool = True, url: str = None) -> None:
         super().__init__(name=name, driver=driver, priority=priority,
                          enabled=enabled)
         self.url = url
         self.es_client = elastic_client
+        try:
+            url_parsed = urlsplit(self.url)
+            self.host = f"{url_parsed.scheme}://{url_parsed.hostname}"
+            self.port = url_parsed.port
+        except Exception as exception:
```

Comment:
What kind of exception can we expect here? Catching general exceptions is usually not a good idea. I know that this is old code, but I'm not sure if this was discussed

Reply:
That's true, but in this case I think the exception could be restricted to ValueError, at least if I'm reading the docs right: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlsplit

Reply:
Changed. Thanks :)